docs/docs/recipes/integrate-logto/express.mdx

```diff
@@ -0,0 +1,167 @@
+---
+sidebar_label: Express
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import redirectUriFigure from './assets/next-redirect-uri.png';
+import AppNote from './fragments/_app-note.mdx';
+import ConfigureRedirectUri from './fragments/_configure-redirect-uri.mdx';
+import FurtherReadings from './fragments/_further-readings.md';
+import GetAppSecret from './fragments/_get-app-secret.mdx';
+import AssumingUrl from './fragments/_web-assuming-url.md';
+import SignInFlowSummary from './fragments/_web-sign-in-flow-summary.mdx';
+
+# Express: Integrate `@logto/express`
+
+<AppNote type="Traditional web" />
+
+## Add dependencies
+
+<Tabs>
+
+  <TabItem value="npm" label="npm">
+
+```bash
+npm i @logto/express cookie-parser express-session
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @logto/express cookie-parser express-session
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```bash
+pnpm add @logto/express cookie-parser express-session
+```
+
+  </TabItem>
+
+</Tabs>
+
+## Init LogtoClient
+
+<AssumingUrl />
+
+<GetAppSecret />
+
+Import and initialize LogtoClient:
+
+```ts
+import { LogtoExpressConfig } from '@logto/express';
+
+const config: LogtoExpressConfig = {
+  appId: '<your-application-id>',
+  appSecret: '<your-app-secret-copied-from-console>',
```

Comment:
nit: Seems we don't need to add `copied-from-console`, as app id is also copied from console. What do you think?